src/app/_components/DistrictSearch/DistrictSearch.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import { cn } from '@utils/cn';
+import { Heading } from '@/app/_components';
+import { getDistrictsList } from '@/app/_components/DistrictSearch/actions';
+import { DistrictList } from '@/app/_components/DistrictSearch/DistrictList';
+
+export async function DistrictSearch({ className }) {
+  const optionsList = await getDistrictsList();
```

Comment:
🟢 do you think its worth creating helper file for 3 lines of code? 🤔

Reply:
getDisctrictList is a function which will be executed on the server. That's why I put it into a separate file.

Reply:
'page.js' will also be executed on server

Reply:
so what's the question?)